velox/functions/prestosql/Reduce.cpp

```diff
@@ -29,7 +29,8 @@ void checkArraySizes(
   const auto* indices = decodedArray.indices();
   const auto* rawSizes = decodedArray.base()->as<ArrayVector>()->rawSizes();
 
-  static const vector_size_t kMaxArraySize = 10'000;
+  const auto maxArraySize =
```

Comment:
Can you set this once in ReduceFunction as a class member?

Reply:
@xiaoxmeng 

ReduceFunction is based on VectorFunction.
The latter does not have initialize() and is stateless, from what I understand, meaning there is only one instance of it in the system.
So, cannot cache that config prop in the ctor.